### PR TITLE
Fb 1897

### PR DESCRIPTION
**Pull request by lorycloutier**
Change datepart function to datetimepart 
FB-1897 
The specs for the datetimepart function are, as far as i can tell, identical to the existing datepart function. Per Pat, replaced the datepart function with datetimepart rather than just adding datetimepart as an alias. Made sure existing tests that were using datepart got switched over.

### CODE REVIEW

File: sql3/test/defs/defs_date_functions.go [hunk 1]
```diff
@@ -24,21 +24,21 @@ var datePartTests = TableTest{
 		{
 			name: "DatePartIncorrectParamsCount",
```
Comment (rachithrr) [inline sql3/test/defs/defs_date_functions.go:25]: Can you change the names as well.

**Review by rachithrr — COMMENTED** [review]
I see fbsql binary pushed here, but i have a feeling that it shouldn't be. 

Looks good otherwise.

**Review by travisturner — DISMISSED** [review]
Please remove the `fbsql` binary from this PR.

Also, there is a make target `make install-fbsql` which will install fbsql into your go/bin (which you should have as part of your path, allowing you to just run `fbsql` at a comand line). You probably shouldn't be building fbsql into the featurebase root directory.

File: sql3/test/defs/defs_date_functions.go (hunk 1 above)
Comment (lorycloutier) [reply sql3/test/defs/defs_date_functions.go]: Fixed. I think the names got added in a different ticket; they weren't there when I was doing the original work.

**Comment by lorycloutier** [comment]
Fixed, that was an accident and I had to look up how to remove it.

**Review by rachithrr — APPROVED** [review]

**Pull request merged**